packages/language-server/src/completion/completions.json

```diff
@@ -213,6 +217,28 @@
+    },
+    {
+      "label": "connectOrCreate",
+      "documentation": "While Prisma Client already provides an `upsert` query, which is available on the top-level of the Prisma Client API, we now also support `upsert`s in a *nested* manner. When writing data with Prisma Client, both in `update` and `create`, you can now provide an `upsert` operation on a related row by using this feature flag.\n\nPlease [share](https://github.com/prisma/prisma-client-js/issues/336) your feedback on how this feature works for you. We are interested in both positive and negative feedback, so we know if this feature is already ready for production! (If encounter any problems, please open a new issue here)."
+    },
+    {
+      "label": "transactionApi",
+      "documentation": "While Prisma already ships transactions within [nested writes](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/relation-queries#nested-writes), there are many use cases, where you might want to perform multiple *unrelated* write operations in a transaction and rollback, if one of them fails.\nBy wrapping your write operations in the new `transaction()` function, you can achieve exactly this!\n\n(Note, that these transactions are not long-running and are executed directly after each other. This is an explicit design decision of Prisma. In case this does not cover your use case yet, please chime in on [GitHub](https://github.com/prisma/prisma-client-js/issues/742)).\nAlternatively, you can store the unresolved promises in variables and pass these to the new transaction function.\n\n\n\nPlease [share your feedback](https://github.com/prisma/prisma-client-js/issues/667) on how this feature works for you. We are interested in both positive and negative feedback, so we know if this feature is already ready for production! (If encounter any problems, please open a new issue [here](https://github.com/prisma/prisma/issues/new/choose))."
+    },
+    {
+      "label": "aggregateApi",
+      "documentation": "While Prisma Client already provides a `count` query for each model, we now introduce further aggregation functionality for number fields.\nIf your model has a field of type `Int` or `Float`, the following aggregations will be available:\n* sum: Sum up all values of a field for the selected rows.\n* min: Get the minimum value for a field for the selected rows.\n* max: Get the maximum value for a field for the selected rows.\n* avg: Get the average value for a field for the selected rows.\n\nPlease [share](https://github.com/prisma/prisma/issues/2838) your feedback on how this feature works for you. We are interested in both positive and negative feedback, so we know if this feature is already ready for production! (If encounter any problems, please open a new issue [here](https://github.com/prisma/prisma/issues/new/choose))."
+    }
+  ],
```

Comment:
Ohhhhhh... that is interesting. I did not expect that we also suggest the values!

Hmm, quite a bit of maintenance at the same time. But probably worth it? @timsuchanek?

Reply:
Should each of them maybe link to the respective feedback issue that are linked here: https://github.com/prisma/prisma/issues/3108?

Reply:
@nikolasburk  Do you mean in addition to the link of the respective issue (e.g. https://github.com/prisma/prisma-client-js/issues/667) or instead of it?

Reply:
We have new, individual feedback issues listed on https://github.com/prisma/prisma/issues/3108 that people should use to leave feedback.

If those links are indeed clickable, that would be nice.

(@nikolasburk We should also go back to the release notes and update these.)